interactive/reporter.go

```diff
@@ -2,17 +2,31 @@ package interactive
 
 import (
 	"fmt"
+
 	"github.com/viktomas/godu/core"
 )
 
-func ReportTree(folder *core.File) []string {
-	report := make([]string, len(folder.Files))
+type Line struct {
+	Text     []rune
+	IsMarked bool
+}
+
+func ReportTree(folder *core.File, marker *core.FileMarker) []Line {
+	report := make([]Line, len(folder.Files))
 	for index, file := range folder.Files {
 		name := file.Name
 		if file.IsDir {
 			name = name + "/"
 		}
-		report[index] = fmt.Sprintf("%s %s", formatBytes(file.Size), name)
+		var marking string
+		isMarked := marker.IsMarked(file)
+		if marker.IsMarked(file) {
+			marking = "*"
```

Comment:
This now fiddles a bit with the 5 char formatting of fileSize introduced in https://github.com/viktomas/godu/pull/22 can we add the `marking` at the end?